sections/Deals/Vesting/VestingDealSection/VestingSection.tsx

```diff
@@ -99,7 +95,7 @@ const VestingDeal: FC<VestingDealProps> = ({
 				header: 'Total Vested',
 				subText: (
 					<>
-						{totalVested ? (
+						{totalVested !== null ? (
```

Comment:
`totalVested !== null` should always be `totalVested != null`

Reply:
Why? This is a good ref for checking null values: https://github.com/lodash/lodash/blob/master/isNull.js#L17